pkg/sidecar_old/server/server.go

```diff
@@ -32,8 +32,8 @@ func NewServer() *Server {
 // Init - configures the server
 func (s *Server) Init(r models.Reader, w models.Writer) {
 	// server requests related
-	s.writeAddr = fmt.Sprintf(":%s", os.Getenv("INSPR_SIDECAR_WRITE_PORT"))
-	s.client = request.NewJSONClient(fmt.Sprintf("http://localhost:%v", os.Getenv("INSPR_SIDECAR_READ_PORT")))
+	s.writeAddr = fmt.Sprintf(":%s", os.Getenv("INSPR_LBSIDECAR_WRITE_PORT"))
+	s.client = request.NewJSONClient(fmt.Sprintf("http://localhost:%v", os.Getenv("INSPR_LBSIDECAR_READ_PORT")))
```

Comment:
maybe lookup these envs before using them

Reply:
As this is the old sidecar architecture and wont be used, I'll just leave as it is